driver/ppm_events_public.h

```diff
@@ -624,7 +634,9 @@ enum ppm_event_type {
 	PPME_CONTAINER_X = 229,
 	PPME_SYSCALL_EXECVE_16_E = 230,
 	PPME_SYSCALL_EXECVE_16_X = 231,
-	PPM_EVENT_MAX = 232
+	PPME_SYSCALL_SIGNALDELIVER_E = 232,
+	PPME_SYSCALL_SIGNALDELIVER_X = 233, /* This should never be called */
```

Comment:
The previous comparison for signal type was wrong. Corrected now and added checks for various signals.
However, it seems from http://www.lehman.cuny.edu/cgi-bin/man-cgi?siginfo.h+3 that "the uncatchable signals SIGSTOP and SIGKILL  have  undefined siginfo codes."
I ave tried the same test as yours by sending SIGKILL to a running process using kill() but the si_pid value is always 0. My reference for struct siginfo is http://lxr.free-electrons.com/source/include/uapi/asm-generic/siginfo.h#L48.